.github/workflows/functional-test-cloud.yaml

```diff
@@ -348,10 +348,48 @@ jobs:
           append: true
           message: |
             :x: Test recipe publishing failed
+  
+  publish-test-bicep-types:
+  name: Publish Radius bicep types to ACR
+    runs-on: ubuntu-latest
+    steps: 
+      - name: Check out code
+        uses: actions/checkout@v4
+      - name: Parse release version and set environment variables
+        run: python ./.github/scripts/get_release_version.py
+      - name: Set up Go ${{ env.GOVER }}
+        uses: actions/setup-go@v5
+        with:
+          go-version: ${{ env.GOVER }}
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '18'
+      - name: Generate Bicep extensibility types from OpenAPI specs
+        run: |
+          make generate-bicep-types VERSION=${{ env.REL_VERSION == 'edge' && 'latest' || env.REL_VERSION }}
+      - name: Upload Radius Bicep types artifacts
+        uses: actions/upload-artifact@v4
+        with:
+          name: radius-bicep-types
+          path: ./hack/bicep-types-radius/generated
+          if-no-files-found: error
+      - name: 'Login via Azure CLI'
+        uses: azure/login@v1
+        with:
+          creds: ${{ secrets.BICEP_TYPES_AZURE_CREDENTIALS }}
```

Comment:
Could you please update the wiki to include information about why we need separate credentials? Do they need to be rotated? If so, do we have a plan to migrate it to workload identity?

Reply:
Sure, the credentials should already be in the wiki but will add more info. The ACR is in the OSS assets subscription (a different sub from the one the tests use) so separate credentials are needed. I think these should be migrated to WI when the rest of the functional tests are